client/app/api/course/LearningMap.js

```diff
@@ -0,0 +1,100 @@
+import BaseCourseAPI from './Base';
+
+export default class LearningMapAPI extends BaseCourseAPI {
+  /**
+   * Fetches all nodes in the learning map for the current user and course.
+   *
+   * @return {Promise}
+   * success response: {
+   *   nodes: Array.<{
+   *     id: string, unlocked: boolean, satisfiability_type: string,
+   *     course_material_type: string, content_url: string,
```

Comment:
Actually I think the json responses should be in camelCase, i.e. the conversion from snake to camel occurs in the jbuilders.

Perhaps we should standardise @ekowidianto

Reply:
Agree! in the future everything should have already been converted to camelCase

Reply:
Created jbuilder file and did the camelCase conversion there.